.github/workflows/mirror-php-package.yml

```
@@ -0,0 +1,39 @@
+name: Mirror PHP package
+description: 'Mirrors a PHP package to a mirror repository to be released to Packagist.org.'
```

actionlint (1.7.7)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)